Sources/GDSCommon/Components/ContentTile/ContentTileView.swift

```diff
@@ -0,0 +1,207 @@
+                bodyLabel.adjustsFontForContentSizeCategory = true
+                bodyLabel.setContentCompressionResistancePriority(.required, for: .vertical)
+                bodyLabel.translatesAutoresizingMaskIntoConstraints = false
+            } else {
+                bodyLabel.isHidden = true
+            }
+            bodyLabel.accessibilityIdentifier = "content-tile-body"
+        }
+    }
+    
+    @IBOutlet private var separatorStack: UIStackView! {
+        didSet {
+            separatorStack.isHidden = !viewModel.showSeparatorLine
+            
+            let separatorView = SeparatorView()
+            separatorStack.addArrangedSubview(separatorView)
+            separatorStack.layoutMargins = UIEdgeInsets(top: 8,
+                                                        left: 16,
+                                                        bottom: 0,
+                                                        right: 0)
+            separatorStack.isLayoutMarginsRelativeArrangement = true
+            separatorStack.accessibilityIdentifier = "content-tile-separator"
+        }
+    }
+    
+    
+    @IBOutlet private var buttonStack: UIStackView! {
+        didSet {
+            buttonStack.spacing = 16
+            buttonStack.layoutMargins = UIEdgeInsets(top: 8,
+                                                     left: 16,
+                                                     bottom: 16,
+                                                     right: 16)
+            buttonStack.isLayoutMarginsRelativeArrangement = true
+            buttonStack.backgroundColor = viewModel.backgroundColour
+            
+            buttonStack.addArrangedSubview(secondaryButton)
+            buttonStack.addArrangedSubview(primaryButton)
+        }
+    }
+    
+    private lazy var secondaryButton: SecondaryButton = {
+        let secondaryButton = SecondaryButton()
+        secondaryButton.accessibilityIdentifier = "content-secondary-button"
+
+        if let view = viewModel as? ContentTileViewModelWithSecondaryButton {
+            secondaryButton.titleLabel?.textColor = .gdsGreen
+            if let icon = view.secondaryButtonViewModel.icon {
+                secondaryButton.symbolPosition = icon.symbolPosition
+                secondaryButton.icon = icon.iconName
+            }
+            secondaryButton.contentHorizontalAlignment = .left
+            secondaryButton.setTitle(view.secondaryButtonViewModel.title.value, for: .normal)
+            secondaryButton.addTarget(self, action: #selector(secondaryButtonTapped), for: .touchUpInside)
+            secondaryButton.isUserInteractionEnabled = true
+            return secondaryButton
+        } else {
+            secondaryButton.isHidden = true
+        }
+        return secondaryButton
+    }()
+    
+    @objc private func secondaryButtonTapped() {
+        (viewModel as? ContentTileViewModelWithSecondaryButton)?.secondaryButtonViewModel.action()
```

Comment:
I feel like this would be better as separate statements rather than one long line.
Perhaps we should use an `if` statement here?